spec/puppet-server-lib/puppet/jvm/http_client_spec.rb

```diff
@@ -21,4 +21,19 @@
       expect(request_options.get_connect_timeout_milliseconds).to equal(connect_timeout)
     end
   end
+
+  context "requests" do
```

Comment:
Ah yeah, I didn't give it any thought but `describe` makes more sense. Will change.